Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/cmd/edit.go

```diff
@@ -294,14 +295,15 @@ outter:
 			}
 
 			if hold, msg := jsonmerge.TestPreconditionsHold(patch, preconditions); !hold {
-				fmt.Fprintf(out, "error: %s", msg)
+				fmt.Fprintf(out, "Error: %s\n", msg)
```

Comment:
ok, updated. Can you also please review the upstream patch at https://github.com/kubernetes/kubernetes/pull/22634